tests/phpunit/elementor/core/admin/test-admin.php

```diff
@@ -10,6 +11,27 @@
 }
 
 class Test_Admin extends Elementor_Test_Base {
+	/**
+	 * @var string
+	 */
+	private $pagenow;
+
+	public function setUp() {
+		global $pagenow;
+
+		parent::setUp();
+
+		$this->pagenow = $pagenow;
+	}
+
+	public function tearDown() {
+		global $pagenow;
+
+		parent::tearDown();
+
+		$pagenow = $this->pagenow;
```

Comment:
It's needed? the original `tearDown` is supposed to restore all globals.

Reply:
Somehow it does not affect `$pagenow`, just checked it again.